plugins/pgbouncer_logs.yaml

```diff
@@ -0,0 +1,84 @@
+version: 0.0.1
+title: PgBouncer
+description: Log parser for PgBouncer
+parameters:
+  - name: file_path
+    type: "[]string"
+    default: "/var/log/pgbouncer/pgbouncer.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    filelog:
+      include:
+        {{ range $fp := .file_path }}
+        - '{{ $fp }}'
+        {{end}}
+      start_at: {{ .start_at }}
+      operators:
+        - type: regex_parser
+          regex: '^(?P<timestamp>\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}.\d{3}(\s*[^\d\s]{3})?)\s*(\[)?(?P<pid>\d+)(\])? (?P<severity>\w+) (?P<message>.*)'
+          severity:
+            parse_from: attributes.severity
+            mapping:
+              critical: fatal
+              info: log
+              notice: stats
```

Comment:
notice isn't a valid alias in opentelemetry like it was in stanza; We've been mapping it to `info2` in general, I think.